src/_Color.sass

```diff
@@ -7,3 +7,4 @@ $color-primary: #9b4dca !default
 $color-secondary: #606c76 !default
 $color-tertiary: #f4f5f6 !default
 $color-quaternary: #d1d1d1 !default
+$color-quinary: #545b61 !default
```

Comment:
the active state color is almost always a variation of the hover state color.
no need to supercharge with a new variable...